scraped_data.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
This is how the output JSON would look like. This file will be deleted in the final merge.

Reply:
Is it possible to write to this file in such a way that there is good indentation on the JSON text?

Something like:
```
[
    {
        "URL": "...",
        "status_tables": [
            {
                "packages": [...],
                "release_states": [...]
            }
        ],
        "notes": [...]
    }
]
```

Reply:
Sure, there should be. I will take a look into it.

Reply:
Can we format this file with json indentation to understand the file format better for review

Reply:
Done